tilelang/tools/bench.py

```
@@ -0,0 +1,144 @@
+        sys.stderr = saved_stderr
+        devnull.close()
+
+
+def process_func(func, *args, repeat=10, warmup=3, **kwargs):
+    # Run a target function multiple times and measure average latency.
+    try:
+        with suppress_output():
+            for _ in range(warmup):
+                func(*args, **kwargs)
+    except Exception:
+        pass
```

Ruff (0.14.5)

`try`-`except`-`pass` detected, consider logging the exception

(S110)